pkg/kv/kvserver/batcheval/cmd_conditional_put.go

```diff
@@ -70,9 +70,12 @@ func ConditionalPut(
 			ReplayWriteTimestampProtection: h.AmbiguousReplayProtection,
 			OmitInRangefeeds:               cArgs.OmitInRangefeeds,
 			OriginID:                       h.WriteOptions.GetOriginID(),
-			MaxLockConflicts:               storage.MaxConflictsPerLockConflictError.Get(&cArgs.EvalCtx.ClusterSettings().SV),
-			TargetLockConflictBytes:        storage.TargetBytesPerLockConflictError.Get(&cArgs.EvalCtx.ClusterSettings().SV),
-			Category:                       fs.BatchEvalReadCategory,
+			// TODO(ssd, msbutler): if args.OriginTimestamp is set, we should use it
+			// here and assert that h.WriteOptions is not set.
+			OriginTimestamp:         h.WriteOptions.GetOriginTimestamp(),
```

Comment:
Should we do this one now?

Reply:
Done. I'll let CI find any tests that i may need to update.